src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -787,6 +850,46 @@ public void AttributedParametersTest()
             Assert.Equal(NullabilityState.NotNull, nullabilityContext.Create(tryGetOutParameters[0]).ReadState);
         }
 
+        public static IEnumerable<object[]> NestedGenericsReturnParameterData()
+        {
+            // public IEnumerable<Tuple<(string name, object? value), int>?> MethodReturnsEnumerableNonTupleNonNonNullValueTupleNonNullNon() => null!;
```

Comment:
nit\style: perhaps a blank line between each of these to help group the comment with the test